its/src/test/kotlin/org/sonarlint/intellij/its/BaseUiTest.kt

```diff
@@ -140,8 +140,8 @@ open class BaseUiTest {
         }
         searchField().text = fileName
         val fileList = jList(JListFixture.byType(), Duration.ofSeconds(5))
-        waitFor(Duration.ofSeconds(5)) { fileList.items.isNotEmpty() }
-        fileList.selectItem(fileList.items[0], false)
+        waitFor(Duration.ofSeconds(5)) { fileList.collectItems().isNotEmpty() }
+        fileList.clickItemAtIndex(0)
         waitFor(Duration.ofSeconds(10)) { editor(fileName).isShowing }
         waitBackgroundTasksFinished()
```

Comment:
A bit of duplication here

Reply:
Indeed, but I think we can improve that later